recipes/barrnap-python/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = 'barrnap' %}
+{% set version = '0.0.5' %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "acac783c1f93e8e9585017be9e128f7f95d8ae77a9ed3b3656fa53d5043e0e21" %}
+
+package:
+  name: barrnap-python
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py2k]
+  skip: True  # [py34]
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - barrnap >=0.8
```

Comment:
So this depends on the original Perl ``barrnap`` tool as packaged here:

https://github.com/bioconda/bioconda-recipes/tree/master/recipes/barrnap

Reply:
Yes;  I have added a clarifying note.  This is to make sure that (1) the actual authors can get accurate usage information of their work, and (2) to ensure that the HMM database is installed in a predictable place.